### PR TITLE
RandPOS: Improve Triggering

### PR DESCRIPTION
**Pull request by MrChrisW**
Hi @SueSmith!

Thought I'd give you a hand and improve the triggers in your new Instant Answer #1473 hope you don't mind :smiley: 

Features: 
+ Use list of triggers avoid using regexp in handle function
+ `map` list of triggers for use with `triggers` function before `handle` 

Let me know if you have any questions. You're welcome to merge this PR and it will be integrated into #1473

:smile_cat:

### CODE REVIEW

File: lib/DDG/Spice/RandPOS.pm [hunk 1]
```diff
@@ -11,26 +12,44 @@ category 'language';
...
 spice to => 'http://api.wordnik.com/v4/words.json/randomWords?includePartOfSpeech=$1&minCorpusCount=10000&limit=$2&api_key={{ENV{DDG_SPICE_WORDNIK_APIKEY}}}&callback={{callback}}';
 spice proxy_cache_valid => '418 1d';
 
-handle remainder => sub {
-
-    return unless $_ =~ /^(noun|nouns|verb|verbs|adjective|adjectives|pronoun|pronouns|preposition|prepositions|conjunction|conjunctions|adverb|adverbs)$/i;
-
-    if($_ =~ /nouns|verbs|adjectives|pronouns|prepositions|conjunctions|adverbs/){
+# List of all trigger words
+my @triggerWords = ('noun', 'nouns',
```
Comment (moollaza) [inline lib/DDG/Spice/RandPOS.pm]: If you wanna get fancy...

```perl
my @triggerwords = map { ($_, $_."s") } qw(noun verb adjective pronoun preposition conjunction adverb)
```

Comment (MrChrisW) [inline lib/DDG/Spice/RandPOS.pm]: @moollaza Sexy! 9cacf11

**Comment by SueSmith** [comment]
Ooh thanks @MrChrisW - I won't pretend I have a clue what you've done here but it's all good learning material for me! 

Merging.. :see_no_evil:

**Comment by MrChrisW** [comment]
@SueSmith :laughing: - perl is great fun to learn! Happy I could help!
Looking forward to seeing your new Instant Answer live :+1:

**Pull request merged**